projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.service.spec.ts

```diff
@@ -1,46 +1,57 @@
 import { Type } from '@angular/core';
 import { async, TestBed } from '@angular/core/testing';
+import { Product, ProductService } from '@spartacus/core';
 import { Observable, of } from 'rxjs';
 import { CmsMerchandisingCarouselComponent } from '../../../cds-models/cms.model';
 import { CdsMerchandisingProductService } from '../../facade/cds-merchandising-product.service';
-import {
-  MerchandisingProduct,
-  MerchandisingProducts,
```

Comment:
I do not have anywhere to put this so I'll just put it here, but can you fix this path, always finish with the ts (may cause production builds failure)

in merchandising-carousel.component.spec.ts

![Screen Shot 2019-12-05 at 1 41 08 PM](https://user-images.githubusercontent.com/29099466/70263688-f0a90a80-1764-11ea-8e98-115151582414.png)

Reply:
Corrected